core/spec/models/spree/permission_sets/dashboard_display_spec.rb

```diff
@@ -10,6 +10,7 @@
 
   context "when activated" do
     before do
+      Spree.Deprecator.warn 'The dashboard_display_spec.rb test is being deprecated.  Please update your source code.'
```

Comment:
Ah, a typo causes the spec error

```suggestion
      Spree.deprecator.warn 'The dashboard_display_spec.rb test is being deprecated.  Please update your source code.'
```